wrappers/csharp/Managed.cs

```diff
@@ -1,16 +1,22 @@
 namespace Devolutions.Cryptography
 {
     using System;
+    using System.Linq;
```

Comment:
Is Linq required just for the .All() in the method SharesLengthAreValid below? If so, a simple for loop would be much faster.

Reply:
Fix